polar_plotter_AREA.py

```diff
@@ -0,0 +1,180 @@
+                        ## for AREA: counting from frequency header, rows 4-40 for azimuthal gain 
+			#if m >= 4 and m <= 40:
+                        if m >= azimuthGainStart and m <= azimuthGainEnd:
+				## Let's just add the azimuthal angles
+                                print(row[4])
+                                azimuth_gain.append(float(row[4]))
+	f.close()
+	### Declare temporary zenith gains
+	#zenith_gains = []
+	### Loop over the zenith angles
+	#for j in range(1, 36):
+	#	## Declare temporary azimuth means
+	#	azimuth_mean = []
+	#	## Loop over the azimuth angles
+	#	k = j*73
+	#	while k < ((j+1)*73):
+	#		azimuth_mean.append(azimuth_gain[k])
+	#		## Remember to increment!
+	#		k += 1
+	#	zenith_gains.append(np.mean(azimuth_mean))
+		## Remember to increment!
+	## Append the list of mean azimuth gains at each zenith to the gains list
+	gains.append(azimuth_gain)
+
+	# Realized gains
+
+	#with open(g.source2 + "/" + "/" + str(i) + "_" + str(g.freq_num) + ".uan", "r") as f:
+	#	## Read in the file
+	#	uan_read = csv.reader(f, delimiter = ' ')
+	#	## Declare list to hold azimuth gains
+	#	azimuth_gain = []
+	#	for m, row in enumerate(uan_read):
+	#		## skip the header
+	#		if m >= 17:
+	#			## Let's just add the azimuthal angles
+	#			azimuth_gain.append(float(row[2]))
+	#f.close()
+	# Declare temporary zenith gains
+	#zenith_gains = []
+	## Loop over the zenith angles
+	#for j in range(1, 36):
+	#	## Declare temporary azimuth means
+	#	azimuth_mean = []
+	#	## Loop over the azimuth angles
+	#	k = j*73
+	#	while k < ((j+1)*73):
+	#		azimuth_mean.append(azimuth_gain[k])
+	#		## Remember to increment!
+	#		k += 1
+	#	zenith_gains.append(np.mean(azimuth_mean))
+	#	## Remember to increment!
+	## Append the list of mean azimuth gains at each zenith to the gains list
+	#realized_gains.append(zenith_gains)
+
+
+print(len(azimuth_gain))
+print(len(gains))
+#print(len(gains[0]))
+#
+#
+#
+## Plotting
+# Make a list for the zenith angles
+zenith_angles = []
+flipped_zenith_angles = [] ## For making a version with realized gains on the other side
+for i in range(0, 37):
+	zenith_angles.append(i*5*np.pi/180)
+	flipped_zenith_angles.append(-i*5*np.pi/180)
+#
+#
+#
+## Declare a figure
+colors = cm.rainbow(np.linspace(0, 1, g.final_antenna - g.initial_antenna + 1))
+
+print(len(zenith_angles))
+
+## Let's make a figure where the lines are overlayed
+fig, ax = plt.subplots(subplot_kw={'projection': 'polar'}, figsize = (10, 8))
+ax.set_theta_zero_location("N")
+ax.set_rlabel_position(225)
+for i in range(0, g.final_antenna - g.initial_antenna + 1):
+	LabelName1 = "{} Gain 1".format(i+g.initial_antenna)
+	LabelName2 = "{} Gain 2".format(i+g.initial_antenna)
+	ax.plot(zenith_angles, gains[i], color = colors[i], linestyle = '--', alpha = 0.4, label = LabelName1, linewidth = 2)
+	#ax.plot(zenith_angles, realized_gains[i], color = colors[g.final_antenna - g.initial_antenna - i], linestyle = ':', alpha = 0.4, label = LabelName2, linewidth = 2)
```

Comment:
```suggestion
for i in range(g.final_antenna - g.initial_antenna + 1):
	label_name_1 = f"{i+g.initial_antenna} Gain 1"
	label_name_2 = f"{i+g.initial_antenna} Gain 2"
	ax.plot(zenith_angles, gains[i], color=colors[i], linestyle='--', alpha=0.4, label=label_name_1, linewidth=2)
	#ax.plot(zenith_angles, realized_gains[i], color=colors[g.final_antenna - g.initial_antenna - i], linestyle=':', alpha=0.4, label=label_name_2, linewidth=2)
```
CamelCase is reserved for classes.